src/dune_pkg/rev_store.mli

```diff
@@ -31,5 +30,11 @@ module Remote : sig
 end
 
 val content_of_files : t -> File.t list -> string list Fiber.t
-val load_or_create : dir:Path.t -> t Fiber.t
+val load_or_create : lock:Dune_util.Flock.t -> dir:Path.t -> t Fiber.t
```

Comment:
I think the lock should stay an implementation detail of `Rev_store`. In other words, we shouldn't mention it in the mli.